src/operators/ai.onnx/ConvTranspose/11/execute_operator__ai_onnx__convtranspose__11__T_tensor_float.c

```diff
@@ -0,0 +1,193 @@
+//this file was generated by ../../../../../../scripts/onnx_generator/OperatorTemplate.py
+#include "operator__ai_onnx__convtranspose__11.h"
+#include "tracing.h"
+#include "utils.h"
+
+//defined in prepare
+int calcOutputSize(int inputSize, int kernelSize, int stride, int dilations, int padStart, int padEnd);
+
+//transformes a 2d pos into a 1d flat array pos
+int calcArrayPos2D(int x, int y, int width) {
+    return y * width + x;
+}
+
+//transformes a 3d pos into a 1d flat array pos
+int calcArrayPos3D(int x, int y, int outputChannel, int width, int height) {
+    return outputChannel * height * width + y * width + x;
+}
+
+//transformes a 4d pos into a 1d flat array pos
+int calcArrayPos4D(int x, int y, int outputChannel, int intputChannel, int width, int height, int nOfOutputChannels) {
+    return intputChannel * nOfOutputChannels *height * width 
+           + outputChannel * height * width 
+           + y * width
+           + x;
+}
```

Comment:
would inline these

Reply:
As far I know is Inline is only a suggestion, if the function is small enough the compiler will inline it even without the inline statement

Reply:
`static inline` is your friend here.
gcc may inline these on its own, but by saying `static inline` you tell gcc to inline them and not generate a symbol, therefore it has to inline these and produces no callable assembler section

Reply:
> You could use `__attribute__((__always_inline__))` if you want force it

this is bad, since it also inlines if `-O0` is set

> only visible in this scope

not really, `static` refers to internal linkage in this translation_unit (file) and that means no symbol, since symbols are needed for external linkage.
please read the c99 standard and gcc manual on this topic if you don't want believe me :)
regarding the combination from current gcc [manual](https://gcc.gnu.org/onlinedocs/gcc/Inline.html)
```
When a function is both inline and static,
if all calls to the function are integrated into the caller,
and the function’s address is never used,
then the function’s own assembler code is never referenced. 
[...]
When an inline function is not static,
then the compiler must assume that there may be calls from other source files;
since a global symbol can be defined only once in any program,
the function must not be defined in the other source files,
so the calls therein cannot be integrated.
Therefore, a non-static inline function is always compiled on its own in the usual fashion. 
```